resources/js/views/settings/users/View.vue

```diff
@@ -361,7 +361,14 @@ export default {
   mounted () {
     EventBus.$on('currentUserChangedEvent', this.togglePermissionFlags);
 
-    if (!['profile', 'view'].includes(this.config.type)) {
+    if (
```

Comment:
the rule is not working correctly. I found many cases where it was loading the data even if not necessary. E.g. why would you want to load the permissions if the user can't edit them?

Reply:
I think this should work
```js
 if (this.config.id === 'new' || (
      PermissionService.can('editUserRole', {
        id: this.config.id,
        model_name: 'User'
      }) && this.config.type === "edit")
    ) {
      this.loadRoles();
    }
```

Reply:
Yeah I've used the wrong method. This should be can!